model/task/db.go

```diff
@@ -194,6 +194,42 @@ var (
 					},
 					"then": evergreen.TaskTimedOut,
 				},
+				// A task will run if it is activated and does not have any blocking deps
+				{
+					"case": bson.M{
+						"$and": []bson.M{
+							{"$eq": []string{"$" + StatusKey, evergreen.TaskUndispatched}},
+							{"$eq": []interface{}{"$" + ActivatedKey, true}},
+							{
+								"$or": []bson.M{
+									{DependsOnKey: 0},
+									{"$eq": []interface{}{"$" + bsonutil.GetDottedKeyName(DependsOnKey, DependencyUnattainableKey), false}},
```

Comment:
Could also do $neq "true" to shorten this up

Reply:
`$ne` *